ssm_jax/hmm/models/mvn_spherical_hmm.py

```diff
@@ -0,0 +1,66 @@
+import jax.numpy as jnp
+import jax.random as jr
+import tensorflow_probability.substrates.jax.bijectors as tfb
+import tensorflow_probability.substrates.jax.distributions as tfd
+from ssm_jax.abstractions import Parameter
+from ssm_jax.hmm.models.base import StandardHMM
+
+
+class MultivariateNormalSphericalHMM(StandardHMM):
+
+    def __init__(self,
+                 initial_probabilities,
+                 transition_matrix,
+                 emission_means,
+                 emission_cov_diag_factors,
+                 initial_probs_concentration=1.1,
+                 transition_matrix_concentration=1.1,
+                 emission_cov_diag_factors_concentration=1.1,
+                 emission_cov_diag_factors_rate=1.1):
+
+        super().__init__(initial_probabilities, transition_matrix, initial_probs_concentration,
+                         transition_matrix_concentration)
+        self._emission_means = Parameter(emission_means)
+        self._emission_cov_diag_factors = Parameter(emission_cov_diag_factors.reshape((-1, 1)),
```

Comment:
For the sake of consistency, I want to ensure that the shape of emission_cov_diag_factors is always `(num_states, 1)`  because a user can specify `emission_cov_diag_factors` in two ways: one with shape `(num_states, )` and shape `(num_states, 1)`.

Reply:
Ah I see. Maybe we should have an assert for ndim == 1 or ndim==2 and shape[1]==1

Reply:
Removed reshaping. Added an assert statement instead.